app/products/calls/components/incoming_calls_container.tsx

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {Platform, StyleSheet, View} from 'react-native';
+import {useSafeAreaInsets} from 'react-native-safe-area-context';
+
+import CallNotification from '@calls/components/call_notification';
+import {useCurrentCall, useGlobalCallsState, useIncomingCalls} from '@calls/state';
+import {
+    CALL_ERROR_BAR_HEIGHT,
+    CURRENT_CALL_BAR_HEIGHT,
+    DEFAULT_HEADER_HEIGHT,
+    JOIN_CALL_BAR_HEIGHT,
+} from '@constants/view';
+
+const topBarHeight = DEFAULT_HEADER_HEIGHT;
+
+const style = StyleSheet.create({
+    wrapper: {
+        position: 'absolute',
+        width: '100%',
+        ...Platform.select({
+            android: {
+                elevation: 9,
```

Comment:
this creates a shadow, is that what we want?

Reply:
Nope, that was not the intention -- removed, thanks.